From patchwork Sun Mar 20 13:21:13 2016
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [U-Boot] sunxi: Enable support for the eMMC found on the orangepi plus
From: Hans de Goede <hdegoede@redhat.com>
X-Patchwork-Id: 599840
Message-Id: <1458480073-22121-1-git-send-email-hdegoede@redhat.com>
To: Ian Campbell <ijc+uboot@hellion.org.uk>
Cc: u-boot@lists.denx.de
Date: Sun, 20 Mar 2016 14:21:13 +0100

This enables support for the eMMC found on the orangepi plus.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 configs/orangepi_plus_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/orangepi_plus_defconfig b/configs/orangepi_plus_defconfig
index c4885f4..770bd98 100644
--- a/configs/orangepi_plus_defconfig
+++ b/configs/orangepi_plus_defconfig
@@ -4,6 +4,8 @@ CONFIG_MACH_SUN8I_H3=y
 CONFIG_DRAM_CLK=672
 CONFIG_DRAM_ZQ=3881979
 CONFIG_DRAM_ODT_EN=y
+CONFIG_MMC0_CD_PIN="PF6"
+CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 # CONFIG_VIDEO is not set
 CONFIG_DEFAULT_DEVICE_TREE="sun8i-h3-orangepi-plus"
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
